src/ui/head.py

```diff
@@ -91,21 +110,27 @@ def ClickBTN(self):
         if self.timer.isActive() == False:
             self.ui.pushButton.setText("Stop Recording")  
             self.timer.start() #start the timer
+
             #start writing the video
             self.ThreadActive = True
-            self.Thread1 = Thread1(self)
+            self.Thread1 = Thread1(self.cameraNumber, self)
+            
             self.Thread1.start()
+            self.ui.output1.append("\nRecording Video "+str(self.cameraNumber)+" Started")
+            
         else: 
             self.ui.pushButton.setText("Start Recording")
             self.timer.stop() 
             self.Thread1.stop()
+            self.ui.output1.append("\nRecording Video "+str(self.cameraNumber)+" Stopped")
+            self.cameraNumber += 1
```

Comment:
Since this is associated to the object via self, this will be a global variable. This will eventually increment outside the range of our cameras.

Reply:
cameraNumber represents the number of videos that are being recorded, not the number of cameras.